common/tests/data/PlayerRatingTests.lua

```diff
@@ -0,0 +1,475 @@
+local PlayerRating = require("common.data.PlayerRating")
+local simpleCSV = require("server.simplecsv")
+local tableUtils = require("common.lib.tableUtils")
+local logger = require("common.lib.logger")
+
+-- If starting RD is too high, or too many matches happen in one rating period, massive swings can happen.
+-- This test is to explore that and come up with sane values.
+local function testWeirdNumberStability() 
+
+  local player1 = PlayerRating(1273, 20)
+  local player2 = PlayerRating(1500, 20)
+
+  local wins = 12
+  local totalGames = 25
+
+  local updatedPlayer1 = player1:newRatingForRatingPeriodWithResults(player1:createSetResults(player2, wins, totalGames), 1)
+  local updatedPlayer2 = player2:newRatingForRatingPeriodWithResults(player1:createSetResults(player1, totalGames-wins, totalGames), 1)
+
+  assert(updatedPlayer1:getRating() > 1073)
+  assert(updatedPlayer2:getRating() < 1500)
+  assert(updatedPlayer2:getRating() > 1338)
+end
```

Comment:
This test seems off.
Not only does it not do what it say it does (instead RD is very small) but the calculation for `updatedPlayer2` seems to create incorrect set results.